packages/frontend/gtag/src/gtag-ga4-events/gtag-ga4-event-add-shipping-info.ts

```diff
@@ -0,0 +1,23 @@
+import { gtagEvent } from "../gtag-event";
+
+import { GtagGa4EventEcommerceAttributes } from "./types";
+
+export interface GtagGa4EventAddShippingInfoParamAttributes extends GtagGa4EventEcommerceAttributes {
+  shipping_tier?: GtagGa4EventAddShippingInfoShoppingTier
+}
+
+export enum GtagGa4EventAddShippingInfoShoppingTier {
```

Comment:
are these pre-approved google values? Just checking they aren't specific to on of our projects?